packages/frontend/app/routes/payments.outgoing.$outgoingPaymentId.tsx

```diff
@@ -0,0 +1,224 @@
+                    outgoingPayment.debitAmount.value,
+                    outgoingPayment.debitAmount.assetScale
+                  ) +
+                    ' ' +
+                    outgoingPayment.debitAmount.assetCode}
+                </p>
+              </div>
+              <div>
+                <p className='font-medium'>Sent Amount</p>
+                <p className='mt-1'>
+                  {formatAmount(
+                    outgoingPayment.sentAmount.value,
+                    outgoingPayment.sentAmount.assetScale
+                  ) +
+                    ' ' +
+                    outgoingPayment.sentAmount.assetCode}
+                </p>
+              </div>
+              <div>
+                <p className='font-medium'>Error</p>
+                {outgoingPayment.error ? (
+                  <p className='mt-1 text-red-500'>{outgoingPayment.error}</p>
+                ) : (
+                  <em>None</em>
+                )}
+              </div>
+              <div>
+                {outgoingPayment.metadata ? (
+                  <>
+                    <button
+                      className='font-medium mb-1 cursor-pointer'
+                      aria-label='toggle metadata visibility'
+                      onClick={() => setShowMetadata(!showMetadata)}
+                    >
+                      {showMetadata ? '▼' : '►'} Metadata
+                    </button>
+                    {showMetadata && outgoingPayment.metadata && (
+                      <pre
+                        className='mt-1 text-sm'
+                        dangerouslySetInnerHTML={{
+                          __html: prettify(outgoingPayment.metadata)
+                        }}
+                      />
+                    )}
+                  </>
+                ) : (
+                  <>
+                    <p className='font-medium'>Metadata</p>
+                    <p className='mt-1'>
+                      <em>None</em>
+                    </p>
+                  </>
+                )}
+              </div>
+            </div>
+          </div>
+        </div>
+        {/* Outgoing Payment General Info - END */}
+
+        {/* Outgoing Payment Liquidity */}
+        <div className='grid grid-cols-1 py-3 gap-6 md:grid-cols-3 border-b border-pearl'>
+          <div className='col-span-1 pt-3'>
+            <h3 className='text-lg font-medium'>Liquidity Information</h3>
+          </div>
+          <div className='md:col-span-2 bg-white rounded-md shadow-md'>
+            <div className='w-full p-4 flex justify-between items-center'>
+              <div>
+                <p className='font-medium'>Amount</p>
+                <p className='mt-1'>
+                  {formatAmount(
+                    outgoingPayment.liquidity ?? '0',
+                    outgoingPayment.receiveAmount.assetScale
+                  )}{' '}
+                  {outgoingPayment.receiveAmount.assetCode}
+                </p>
+              </div>
```

Comment:
This displays the outgoing payment liquidity amount using the `receiveAmount` asset code and scale. I wasn't entirely sure if this was the right asset to base it on (there is also `sentAmount` and `debitAmount`).

Reply:
All amounts (`receiveAmount`, `sentAmount` and `debitAmount`) are present when fetching an outgoing payment, therefore it should not matter which `assetScale` and `assetCode` are used when displaying the liquidity amount (since a wallet address is tied with an asset).

Reply:
It should be the _sender_-facing asset: which is either debit or sent amount. If the outgoing payment is sending USD -> EUR, debit and sent would be in USD, and that is the asset of which liquidity would get withdrawn from.